examples/layouts/flex/lv_example_flex_1.c

```diff
@@ -28,7 +28,7 @@ void lv_example_flex_1(void)
         lv_obj_set_size(obj, 100, LV_PCT(100));
 
         label = lv_label_create(obj);
-        lv_label_set_text_fmt(label, "Item: %"LV_PRIu32, i);
+        lv_label_set_text_fmt(label, "Item: %"LV_PRIu32"", i);
```

Comment:
should be:
```c
 lv_label_set_text_fmt(label, "Item: %" LV_PRIu32, i);
```

Reply:
This is cherry-picked from commit 3b7411893f35e9891569f3423e0aae55d1305724 which is already merged into the master branch. Actually, these three styles have the same behavior. Is there any coding style that should be followed for this part in the LVGL project?  Seems that it's not identical throughout the project. Will revert this part back.